app/src/main/java/com/nervousfish/nervousfish/activities/CreateProfileActivity.java

```diff
@@ -24,23 +25,24 @@
 
 import cn.pedant.SweetAlert.SweetAlertDialog;
 
+import static com.nervousfish.nervousfish.modules.constants.Constants.ExplicitFieldResultCodes.ALl_FIELDS_EMPTY;
+import static com.nervousfish.nervousfish.modules.constants.Constants.ExplicitFieldResultCodes.GOOD_FIELD;
+import static com.nervousfish.nervousfish.modules.constants.Constants.ExplicitFieldResultCodes.NAME_EMPTY;
+import static com.nervousfish.nervousfish.modules.constants.Constants.ExplicitFieldResultCodes.PASSWORDS_NOT_EQUAL;
+import static com.nervousfish.nervousfish.modules.constants.Constants.ExplicitFieldResultCodes.PASSWORD_EMPTY;
+import static com.nervousfish.nervousfish.modules.constants.Constants.ExplicitFieldResultCodes.PASSWORD_TOO_SHORT;
+import static com.nervousfish.nervousfish.modules.constants.Constants.InputFieldResultCodes.EMPTY_FIELD;
+import static com.nervousfish.nervousfish.modules.constants.Constants.InputFieldResultCodes.TOO_SHORT_FIELD;
+
+
 /**
  * The {@link android.app.Activity} that is used to create a user profile when the app is first
- * used.
+ * used. Suppresses return count to allow multiple returncodes while checking input fields.
```

Comment:
Can you add the comment on the suppression after the `@SuppressWarnings` statement, just like it is done everywhere in the repo 😇